nightingale_manipulation/src/nightingale_manipulation/manipulation_action_client.py

```diff
@@ -70,12 +67,6 @@ def __init__(self):
         )
         self.right_arm.wait_for_server()
 
-        # TODO: replace with service call
-        self.right_arm_joint_names = CFG["right_arm_home"]["names"]
-        self.left_arm_joint_names = CFG["left_arm_home"]["names"]
```

Comment:
I need these to be members of the ManipulationJointControl class, otherwise they'll need to be passed into the cmd_left/right_arm functions (and update_joint_state function in the future). These values could be passed into the ManipulationJointControl class on initialization so that the config service doesn't have to run in this class.